npm-shrinkwrap.json

```diff
@@ -5,8 +5,8 @@
   "node-version": "v0.10.38",
   "dependencies": {
     "babel-core": {
-      "version": "5.6.5",
-      "resolved": "https://registry.npmjs.org/babel-core/-/babel-core-5.6.5.tgz",
+      "version": "5.6.12",
```

Comment:
This happened again? Can we stop updating libraries in PRs, where it does not make sense?